docs/getting-started/installation.rst

```diff
@@ -58,7 +58,7 @@ The following are a list of optional dependencies and their purpose:
 +===================================+=========================================+
 | ``matplotlib``                    | Using Colormaps                         |
 +-----------------------------------+-----------------------------------------+
-| ``PyQt5==5.11.3``                 | Background plotting                     |
+| ``pyvistaqt``                     | Background plotting                     |
```

Comment:
We should take this out of "dependencies" and just link to it as an offshoot project.

Reply:
Done, and added a reference to `pyvistaqt` in our plotting docs.